.github/workflows/release.yml

```diff
@@ -23,8 +23,14 @@ jobs:
           echo "VERSION=$version" >> $GITHUB_ENV
           if [[ "$version" == *"-rc"* ]]; then
             echo "IS_RC=true" >> $GITHUB_ENV
+            # Remove the patch version and `-rc` suffix
+            base_version=$(echo $version | sed -E 's/([0-9]+\.[0-9]+)\.[0-9]+-rc[0-9]+/\1./')
+            current_date=$(date +"%Y%m%d%H")
```

Comment:
Question: If we want to make a release within an hour of previous release (unlikely), we won't be able to? We will have to wait for upto an hour to make the next release.

Reply:
I thought about it last week and I think yes, we won't be able. But as you said it's unlikely, we could change on the fly if at some point we deadly need to do multiple pre-release within an hour, but that would be very strange.